tests/functional/centos/7/docker-cluster/group_vars/all

```diff
@@ -0,0 +1,25 @@
+---
+
+ceph_stable: True
+mon_containerized_deployment: True
+osd_containerized_deployment: True
+mds_containerized_deployment: True
+rgw_containerized_deployment: True
+ceph_mon_docker_interface: eth1
+ceph_mon_docker_subnet: "192.168.15.0/24"
+ceph_osd_docker_extra_env: "CEPH_DAEMON=OSD_CEPH_DISK_ACTIVATE,OSD_JOURNAL_SIZE=100"
+ceph_docker_on_openstack: False
+public_network: "192.168.15.0/24"
+cluster_network: "192.168.16.0/24"
+journal_size: 100
+journal_collocation: true
```

Comment:
@leseb We've always had 2 variables since I've joined. :) I'm not sure why either. I think it's for the same reason many things are split between containerized and non-containerized but don't necessarily have to be that way. I agree we should consolidate to `devices`.